decidim-proposals/app/views/decidim/proposals/admin/proposals/bulk_actions/_scope-change.html.erb

```diff
@@ -6,11 +6,7 @@
       <% end %>
     </div>
 
-    <%= scopes_picker_tag(
-          :scope_id,
-          try(:current_participatory_space)&.scope,
-          field: t("models.proposal.fields.scope", scope: "decidim.proposals")
-        ) %>
+    <%= scopes_select_tag(:scope_id, root: nil) %>
```

Comment:
I am not 100% this is a bug, but if we accept this change, you would be able to have a scope defined on the participatory space, yet, you could assign the projects to any Scope available in the system.

This is a different behavior than what we currently have on the project form, which is actually validating so that the scope is a child of the participatory process scope.

Reply:
Fixed, the default behavior is to use the current participatory space scope as root on the scopes select, I've removed the nil root argument